#44

raunakdoesdev opened the pull request:
Fixes Issue #44

raunakdoesdev committed:
Fixed directory path and made logs continuously update.

raunakdoesdev committed:
Merge remote-tracking branch 'upstream/master'

raunakdoesdev committed:
Merge remote-tracking branch 'upstream/master'

raunakdoesdev committed:
Remove duplicate appendature of save/ when saving.

raunakdoesdev committed:
Fixed data ctr problem w/ end of epoch.

raunakdoesdev committed:
Run autopep8 -iaar .

raunakdoesdev committed:
Revert "Fixed data ctr problem w/ end of epoch."

This reverts commit 75a2990a0fc8589f6ea2b7f31f39db3a84772e48.

raunakdoesdev committed:
Revert "Run autopep8 -iaar ."

This reverts commit ac6fc2f5debdab309dc9c02a9d954f10a553c92d.

raunakdoesdev committed:
Ran autopep8 -iaar .

raunakdoesdev committed:
gacp

raunakdoesdev committed:
Removed debug statements.

raunakdoesdev committed:
Autopep8

raunakdoesdev committed:
Removed duplicate .weights

tpankaj commented on Train.py:
```diff
@@ -62,6 +62,10 @@ def run_net(data_index):
                 # Logging Loss
                 epoch_train_loss.add(data.train_index.ctr, batch.loss.data[0])
+                epoch_train_loss.export_csv(
+                    'logs/epoch%02d_train_loss.csv' %
```
Is it going to be a good idea to re-export after every batch? Maybe it should be changed to only writing one line at a time rather than overwriting the whole file.

raunakdoesdev replied:
In my experiments this doesn't really affect training speed (still getting 200-250 Hz). I think a better idea would be to move this in with the if ARGS.display stuff, that way it's updated less frequently (user can specify).

raunakdoesdev commented:
I realized this also fixes #26

tpankaj reviewed (changes requested):
All looks good except my concern in the comment about csv exporting.

raunakdoesdev committed:
Remove trailing whitespace Data.py

raunakdoesdev committed:
Fixed Tushar's concern about rate of csv writing.

Also added print timer checking for validation. (Don't want it printing
too much and destroying your buffer)

raunakdoesdev committed:
Merge branch 'master' into #44

tpankaj reviewed (approved):
You also fixed Issue #28.
